scripts/zones/Meriphataud_Mountains_[S]/mobs/Bloodlapper.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area:
+-- NPC:  Yilbegan
+-----------------------------------
+
+require("scripts/globals/titles");
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:addMod(MOD_STORETP, 25); 
+    mob:addMod(MOD_ATT, 50);
```

Comment:
should be in onMobInitialize (check luautils for the correct spelling, cause lolUS engrish)